user/package.json

```diff
@@ -19,6 +19,7 @@
     "@material-ui/core": "^4.11.3",
     "@material-ui/icons": "^4.11.2",
     "axios": "^0.21.1",
+    "fontsource-roboto": "^4.0.0",
```

Comment:
The defined font family for the project is Nunito and was already installed. Delete roboto by running the following command in the project:

```bash
npm uninstall -S fontsource-roboto
```